api/src/main/resources/messages_pt.properties

```diff
@@ -1595,7 +1595,7 @@ PersonAttributeType.UnretirePersonAttributeType=Reactivar Tipo de Atributo Pesso
 PersonAttributeType.UnretirePersonAtrributeMessage=Tem certeza de que quer reactivar este objeto? Ela irá entăo ser restaurado para o sistema
 PersonAttributeType.java.util.Date.warning=Um <i>bug</i> nas versőes mais antigas do OpenMRS permitiu escolher java.util.Date, o que causou dados a serem armazenados em formato de data do usuário. Isso significa que se alguém gravou uma data em dd/mm/aaaa e tentar vę-lo ou editá-lo no formato mm/dd/aaaa entăo as coisas văo quebrar. Se vocę está vendo esta mensagem, entăo vocę tem um tipo de atributo Pessoa em seu banco de dados cujo formato está definido para java.util.Date. <br/> Vocę precisa alterar manualmente todos os dados armazenados na tabela person_attribute para este tipo para yyyy-mm-dd formato, e depois mudar o formato deste tipo de atributo Pessoa a ser um AttributableDate.
 
-PersonAttributeType.Mother'sName=Nome da Măe
+PersonAttributeType.MothersName=Nome da Măe
```

Comment:
@MarcusViniciusCavalcanti after taking a second look at this, the actual message key in use is what you changed. So to switch to this new key, you would need to first search and replace all places where the old key was used. Am therefore closing this pull request such that you create a new one with this recommendation. Thanks again for taking time to work on this! 👍